src/components/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+type HeaderProps = {
+  title: string;
+};
+
+class Header extends React.Component<HeaderProps, object> {
+  constructor(props: HeaderProps | Readonly<HeaderProps>) {
+    super(props);
+  }
+  render() {
+    const path = window.location.pathname.slice(1) + '';
```

Comment:
accessing window directly in a functional component is generally not recommended

You can use hook useLocation from react-router-dom